setup.py

```diff
@@ -54,6 +54,7 @@
     'opencv': ['opencv-python>=3.2.0.6'],
     'tf': ['tensorflow>=1.14.0'],
     'tf_gpu': ['tensorflow-gpu>=1.14.0'],
+    'pyspark': ['pyspark>=2.1.0'], 
```

Comment:
I am not sure but adding pyspark as a dependency might be problematic, I think it might install Pyspark from pip when we build the docker image but then we also mount pyspark somewhere ourself. I suspect that was also the reason why Kim removed it in the first place, because back then we were still adding pyspark as a zip file manually.

Reply:
Good catch. Is `hops-petastorm` supposed to be only used on hopsworks instances? If so, do we need the tf and tf_gpu as well? They conflict with the current TF version in Hopsworks

Reply:
nevermind, didn't see it's in the extra_requires because github was hiding it.

Reply:
didn't investigate much but leaving without pyspark': ['pyspark>=2.1.0'] it installs pyspark 3.0.0 and thus messes with conda env. Adding this worked

Reply:
In which conda environment? the one inside docker containers, or some other local one, that you were testing with?
I think unless you do `pip install hops-petastorm[pyspark]` it shouldn't install any pyspark as dependency since it is in the `extras_require` dependencies. and in the conda base env we install hops-petastorm==0.7.6, not with `[pyspark]`. So I believe it is correct the way it is. We just need to bump the version here https://github.com/logicalclocks/docker-images/blob/b776a2bf9951beb656df5e74722d9a82f8613c3c/base_image/python36/env.yml#L21once the new 0.9.4 version is published to pypi.

Reply:
I had issue when I did `pip install petastorm==0.9.4` in the base image. pip install from davitbzh:v0.9.4, i.e cherry picking our commits from hops_petastorm worked well

Reply:
@davitbzh let's merge the petastorm PR (which one? I see two) then we publish hops-petastorm on pypi and meanwhile you can submit the docker-images PR to bump the version

Reply:
ok I see this is the one we want https://github.com/logicalclocks/petastorm/pull/2
we can merge both actually